examples/with-tailwind/tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+const colors = require("tailwindcss/colors");
```

Comment:
What provides this? I don't see it in `/package.json` (and, arguably, it doesn't really belong there).

Reply:
Moved the config to its own package as suggested, and added the correct dependency to the package